src/Entities/Project.php

```diff
@@ -37,6 +57,27 @@ public function getIsOverdue(): ?bool
         return $this->isOverdue;
     }
 
+    public function toAssociativeArrayAllProperties(): array
+    {   
+        $usersArray = $this->users;
```

Comment:
Rayna prefers just 'users' rather tahn users array - and also, given that this is only used once, could it just be `foreach ($this->users as $user)`?